.github/workflows/push-image.yaml

```diff
@@ -0,0 +1,82 @@
+---
+name: Build and push image to ECR
+
+on:
+  workflow_dispatch:
+    inputs:
+      git_sha:
+        description: "Enter the git sha which you would like to build"
+        required: true
+        type: string
+        default: 42d2005d04b59d14aacf4eef5ccee353e9a531b7
```

Comment:
I'm wondering if this parameter is really needed? :thinking:  It seems it is only used to create the name of the image that is pushed, it is not used to checkout the specific commit for build... If that's the case, the current commit hash can be easily derived using: `git rev-parse --short HEAD` (gives shortened version already)